<?xml version="1.0" encoding="UTF-8"?>
<TestRun id="7336dfd6-b47a-44bd-87bb-b7eb004971ca" name="Jeff@MEGATRON 2010-05-20 23:29:42" runUser="Megatron\Jeff" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <TestSettings name="Local" id="56af038c-9556-44d9-ab8d-01898a7ceee6">
    <Description>These are default test settings for a local test run.</Description>
    <Deployment enabled="false" runDeploymentRoot="Jeff_MEGATRON 2010-05-20 23_29_42">
      <DeploymentItem filename="C:\Users\Jeff\Documents\My Dropbox\JeffDev\JeffDev\Dev\MS.Net\LaunchLater\LaunchLaterOM\bin\Debug\LaunchLaterOM.dll" />
    </Deployment>
    <Execution>
      <TestTypeSpecific />
      <AgentRule name="Execution Agents">
      </AgentRule>
    </Execution>
  </TestSettings>
  <Times creation="2010-05-20T23:29:42.0288011-05:00" queuing="2010-05-20T23:29:42.0538026-05:00" start="2010-05-20T23:29:42.1008053-05:00" finish="2010-05-20T23:29:42.6778383-05:00" />
  <ResultSummary outcome="Failed">
    <Counters total="1" executed="1" error="0" failed="1" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" passed="0" completed="0" inProgress="0" pending="0" />
  </ResultSummary>
  <TestDefinitions>
    <UnitTest name="TestApplicationNotFound" storage="c:\users\jeff\documents\my dropbox\jeffdev\jeffdev\dev\ms.net\launchlater\launchlaterunittests\bin\debug\launchlaterunittests.dll" id="31d7f813-d586-5d82-8ce6-b558f16b8407">
      <Execution id="0f7685c0-5803-454f-b9ef-cf3f1f4be573" />
      <TestMethod codeBase="C:/Users/Jeff/Documents/My Dropbox/JeffDev/JeffDev/Dev/MS.Net/LaunchLater/LaunchLaterUnitTests/bin/Debug/LaunchLaterUnitTests.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="LaunchLaterUnitTests.LLApplicationManagerTests, LaunchLaterUnitTests, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="TestApplicationNotFound" />
    </UnitTest>
  </TestDefinitions>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <TestEntries>
    <TestEntry testId="31d7f813-d586-5d82-8ce6-b558f16b8407" executionId="0f7685c0-5803-454f-b9ef-cf3f1f4be573" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <Results>
    <UnitTestResult executionId="0f7685c0-5803-454f-b9ef-cf3f1f4be573" testId="31d7f813-d586-5d82-8ce6-b558f16b8407" testName="TestApplicationNotFound" computerName="MEGATRON" duration="00:00:00.0444545" startTime="2010-05-20T23:29:42.1518082-05:00" endTime="2010-05-20T23:29:42.6408361-05:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="0f7685c0-5803-454f-b9ef-cf3f1f4be573">
      <Output>
        <ErrorInfo>
          <Message>Test method LaunchLaterUnitTests.LLApplicationManagerTests.TestApplicationNotFound did not throw expected exception LaunchLaterOM.LLApplicationNotFoundException.</Message>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
  </Results>
</TestRun>